src/index.js

```diff
@@ -533,17 +533,45 @@ export default class RNPickerSelect extends PureComponent {
         );
     }
 
+    renderWeb() {
```

Comment:
seems like we could just share the `renderAndroidNativePickerStyle` method and rename it? looks almost identical

Reply:
Yeah, fair enough we could. I considered doing that originally, though I didn't know if we might want to have it separated for further changes in the long term. I'm totally happy with having it as a single component

Reply:
What possible names do you think are a good idea? I'm trying to think of an idea that encapsulates both "native Android picker" and "web".

Reply:
Alternatively, since the web doesn't need the placeholder styling, we could have a separate component. It's your library so it's really up to you.

Reply:
I uploaded a new PR with changes to the web component. I don't know if it's enough to justify a separate component or not. The only reason I can think to have it is because a user might want different Android and web styles